control-plane/tests/test_service_persistence_ingest_case_lifecycle.py

```
@@ -19,6 +19,110 @@
 
 
 class IngestCaseLifecyclePersistenceTests(ServicePersistenceTestBase):
+    def test_service_delegates_detection_lifecycle_operations(self) -> None:
+        store, _ = make_store()
```

Ruff (0.15.10)

[error] Undefined name `make_store`

(F821)